zcncore/transaction_query.go

```diff
@@ -399,3 +402,51 @@ func (tq *TransactionQuery) GetConsensusConfirmation(ctx context.Context, numSha
 
 	return confirmationBlockHeader, confirmationBlock, lfbBlockHeader, nil
 }
+
+func (tq *TransactionQuery) GetInfo(ctx context.Context, numSharders int, query string) (*QueryResult, error) {
+
+	consensuses := make(map[uint64]int)
+	var maxConsensus int
+	var consensusesResp QueryResult
+	// {host}{query}
+	err := tq.FromAll(ctx, query,
+		func(qr QueryResult) bool {
+			if qr.StatusCode != http.StatusOK {
+				return false
+			}
+
+			hash := fnv.New64a()
+			hash.Write(qr.Content)
+			h := hash.Sum64()
+			consensuses[h]++
```

Comment:
Sharders could be on  different latest finalized block when they receive the request, hence will give different results even they are all working well and synced. We may need to return the latest finalized block info, such as round number and block hash in each sharder's response, so clients can determinte which one can be used. Before that, comparing the content hash may not work.

Reply:
@peterlimg  it is fixing bug on https://github.com/0chain/gosdk/blob/a8ac8468109d46a561df4639cdf5bf20d42a3c87/zcncore/wallet.go#L820.  should `GetInfo` be executed with the same logic as `verify`?

Reply:
there are two sharders (01 and 02). in old `getInfoFromSharders`, it will always fails when sharder01 is down, and sharder02 works. Because sharder01 gives 502 first, and sharder02 gives 200. so resultMap[502] and resultMap[200] are both 1. but 502 comes first. so it is last result.